hotspots/calculation.py

```diff
@@ -24,7 +24,11 @@
 import time
 from functools import reduce
 from os import system, environ
-from os.path import join
+from os.path import join, dirname
+
+ghecom =join(dirname(dirname(__file__)),"win_ghecom/Ghecom.exe")

```

Comment:
I'd probably add some info about the requirements to the Fhecom.exe file naming and location to README

Reply:
I think the README needs a lot of work to be honest as it looks like it is very out of date on how to get it working.